pages/signInPage.tsx

```diff
@@ -0,0 +1,121 @@
+                    <path d="M31.768 5.33337H7.98463C6.5246 5.33337 5.34204 6.52671 5.34204 8.00004V32C5.34204 33.4734 6.5246 34.6667 7.98463 34.6667H31.768C33.228 34.6667 34.4106 33.4734 34.4106 32V8.00004C34.4106 6.52671 33.228 5.33337 31.768 5.33337ZM14.5303 29.3334H10.6325V16.6774H14.5303V29.3334ZM12.5418 14.868C11.2852 14.868 10.2692 13.84 10.2692 12.5747C10.2692 11.3094 11.2866 10.2827 12.5418 10.2827C13.7944 10.2827 14.8131 11.3107 14.8131 12.5747C14.8131 13.84 13.7944 14.868 12.5418 14.868ZM29.1307 29.3334H25.2355V23.1787C25.2355 21.7107 25.2091 19.8227 23.2099 19.8227C21.1817 19.8227 20.8699 21.4214 20.8699 23.072V29.3334H16.9747V16.6774H20.714V18.4067H20.7669C21.2874 17.412 22.5585 16.3627 24.4546 16.3627C28.4013 16.3627 29.1307 18.984 29.1307 22.392V29.3334Z" fill="white" />
+                  </a>
+                </svg>
+              </div>
+              <div>
+                <svg width="40" height="40" viewBox="0 0 31 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <a href="https://www.instagram.com/utdais/?hl=en">
+                    <path d="M9.85703 1.80005C5.59386 1.80005 2.12744 5.29805 2.12744 9.60005V20.4C2.12744 24.702 5.59386 28.2 9.85703 28.2H20.5595C24.8227 28.2 28.2891 24.702 28.2891 20.4V9.60005C28.2891 5.29805 24.8227 1.80005 20.5595 1.80005H9.85703ZM22.3433 6.60005C22.9973 6.60005 23.5325 7.14005 23.5325 7.80005C23.5325 8.46005 22.9973 9.00005 22.3433 9.00005C21.6892 9.00005 21.1541 8.46005 21.1541 7.80005C21.1541 7.14005 21.6892 6.60005 22.3433 6.60005ZM15.2083 8.40005C18.8174 8.40005 21.7487 11.358 21.7487 15C21.7487 18.642 18.8174 21.6 15.2083 21.6C11.5992 21.6 8.66786 18.642 8.66786 15C8.66786 11.358 11.5992 8.40005 15.2083 8.40005ZM15.2083 9.60005C12.2591 9.60005 9.85703 12.024 9.85703 15C9.85703 17.976 12.2591 20.4 15.2083 20.4C18.1574 20.4 20.5595 17.976 20.5595 15C20.5595 12.024 18.1574 9.60005 15.2083 9.60005Z" fill="white" />
+                  </a>
+                </svg>
+              </div>
+            </div>
+            <div>
+            </div>
+          </div>
+          <div className="bg-white w-screen h-screen right-0 relative">
+            <div className="flex flex-col justify-center items-center pt-96">
+              <p className="text-black text-2xl font-bold tracking-wide">
+                Sign In/ Sign <br />
+                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Up
+              </p>
+              <a onClick={() => signIn("cognito")}>
+                <div className="flex flex-row rounded border border-gray-400 mt-2">
+                  <div className="ml-4 mt-2">
+                  <Image
+                    src="/gr.png"
+                    alt="AIS logo"
+                    width={35}
+                    height={40}
+                  />
+                  </div>
+                  <p className="m-4 mr-8 pr-3 font-bold tracking-wide text-lg text-gray-400">Sign In with Google</p>
```

Comment:
For sign in with google use the official brand assets provided by google for a consistent look and feel. 

[Brand Assets](https://developers.google.com/identity/branding-guidelines)